### DISCUSSION TITLE
Customising existing alerts

### DISCUSSION THREAD

**Discussion by VincenzoDo** [prometheus-operator/kube-prometheus#1576] [Q&A]
Hello,

how can I change the expression of an existing alert?

If I look at the examples, and try to use the update part of https://github.com/prometheus-operator/kube-prometheus/blob/main/examples/changing-default-rules.libsonnet the rules don't get updated as I would expect. Also there are no changes in the manifests.

Did anyone manage to update a rule?

Thanks,

Vincenzo

**Comment by alondahari** [comment]
Make sure you don't miss the last line of the example file:
`{ ['exampleApplication-' + name]: kp.exampleApplication[name] for name in std.objectFields(kp.exampleApplication) }`
